### DISCUSSION TITLE
сила+урон оружия=Баланс

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#353] [General]
Почитал на  WOP  про Шпагу Мастера. Как играющий через шпагу напишу пару строк.

- Силач в 1 главе **не сможет** одеть Шпагу Мастера не вкладываю ЛП в ловкость. 
- Силач **Мент** сможет одеть во 2 главе  Шпагу Мастера не вкладываю ЛП в ловкость. 
- Силач **не_мент** сможет одеть во 2 главе  Шпагу Мастера не вкладываю ЛП в ловкость, если пойдёт в Миненталь или Ярик за доп бонусами
- Силач может одеть в 1 главе Рапиру без прокачки ловкости за ЛП, если соберёт **все бонусы**, в т.ч. скрижали  которые есть в Хориносе.

Урон в Готике идёт рывками(порогами). одним из порогов является 180 урона. при 180 ущербе можно убивать луркеров/снепперов/краулеров с одного удара. при этом если у героя 185-190-195 урона он фактически ни чем не отличается от 180 урона. 

180 ущерба для силача в 1 главе это не только "ловкостная" Рапира, но и различное брутально-силовое оружие. для наглядности сделал табличку.

Название  | урон | Бонус к 1р | требует | сила | ущербе 
-- | -- | -- | -- | -- | --
Рапира | 100 | 8% | 50 ловк | 80 | 180
Machete | 100 | 3% | 90 силы | 90 | 190
Rubinklinge | 100 | 7% | 90 силы | 90 | 190
Bartaxt | 90 |   | 90 силы | 90 | 180
Doppelaxt | 90 |   | 90 силы | 90 | 180
Morgenstern* | 90 |   | 90 силы | 90 | 180

_*имеет дополнительных 50 ущерба против големов/каменных стражей и т.п._

Таким образом для силача ловкостное оружие является не "панацеей";-)  а вот что бы собрать 40 бонусов ловкости в 1 главе, нужно приложить не только серьезные усилия, но ещё и потратить раньше времени 15 ЛП на изучения чтения скрижалей. Возможно большинству покажется проще вложить не 15 ЛП в языке и бегать собирать 10 ягодок гоблина по всему Хоринису, а просто вложить 20ЛП в силу(90 силы кстати так же позволяет поднять арбалет на 120/60).

Наверное можно было бы, поговорить и про вторую главу, но там ситуация особо не поменяется. Мент конечно же может в начале второй главы поднять шпагу одев амулет на ловкость, но это никак не отменяется того что ему нужно будет всё равно собирать бонусы к ловкости и с чем то(с Рапирой?) ходить всю 1 главу.

Основную возможность которую даёт Шпага Мастера это максимально рано выйти на высокий процент навыка 1р. Но в тоже время вместо кольца на ловкость для Шпаги мастера можно одеть кольцо на навык к 1р(10%) которое есть в Яркендаре. Мне например, при игре за Наёмник, что бы снять второе кольцо на ловкость и одеть вместо него кольцо на навык 1р, пришлось превратится в снеппера и залезть в сокровищницу дракона в Минентале(скрижаль +4 к ловкости и ягода гоблина +1 к ловкости). а что бы снять пояс на ловкость, пришлось учить скрижали 3 уровня(15 лп) и дать одежду отшельнику(+6 к ловкости). В моей игре конечно это всё оправдано, так как позволяет выйти на 100% 1р в начале Ярика и иметь 180 урона. а мне от 100% психологически спокойно:-)

*****

Если говорить о каких то серьезных монстрах и поздних главах, то вот цифры

Орк | Варг | Ищущий | Драконий Снеппер | Элитный Орк | Человек Ящер
-- | -- | -- | -- | -- | --
  |   | 5x210 |   |   | 3 x 217
4 x 225 | 3 x 225 |   |   |   |  
  |   | 4x230 |   |   |  
3 x 250 |   |   | 4 x 250 | 5 x 250 |
  |   |   |   |   | 2 x 260
  |   | 3 x 264 |   |   |  
  | 2 x 275 |   |   | 4 x 273 |  
  |   |   | 3 x 284 |   |  
2 x 300 |   |   |   |   |  
  |   |   |   | 3 x 310 |  
  |   | 2 x330 |   |   |  
  |   |   | 2 x 350 |   |  
  |   |   |   | 2 x 385 |  
  |   |   |   |   | 1 x 390

**225 / 230  урона** 

Название  | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Шпага Мастера | 120 | 10% | 60 ловк | 105/110 | 225/230
El Bastardo | 120 | 8% | 110 силы | 110 | 230

**250  урона**

Название  | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Шпага Мастера | 120 | 10% | 60 ловк | 130 | 250
El Bastardo | 120 | 8% | 110 силы | 130 | 250
Folteraxt  | 125 | - | 125 силы | 125 | 250

**260 / 275  урона**

Название  | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Шпага Мастера | 120 | 10% | 60 ловк | 140 / 155 | 260 / 275
El Bastardo | 120 | 8% | 110 силы | 140 / 155 | 260 / 275
Folteraxt  | 125 | - | 125 силы | 135 / 150 | 260 / 275
Betty  | 130 | - | 110 ловкости| 130 / 145 | 260 / 275
Orkschlächter | 130 | - | 130 силы | 130 / 145  | 260 / 275
Erz-Schlachtklinge*| 140 | 10% | 120 силы | 120 / 135  | 260 / 275

**300 / 310  урона**

Название | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | -- 
Шпага Мастера | 120 | 10% | 60 ловк | 180 / 190 | 300 / 310
El Bastardo | 120 | 8% | 110 силы | 180 / 190 |  300 / 310
Folteraxt | 125 | - | 125 силы | 175 / 185 |  300 / 310
Betty | 130 | - | 110 ловкости | 170 / 180 |  300 / 310
Orkschlächter | 130 | - | 130 силы | 170 / 180 |  300 / 310
Erz-Schlachtklinge* | 140 | 10% | 120 силы | 160 / 170 |  300 / 310

*учит Беннет в финале 3 главы, доступен только ОНД

вот эти таблички наглядно показывают, что если силач решил вложить ЛП в ловкость, что бы одеть шпагу мастера, потому что Шпага Мастера "имба", то ему этот 1-3 уровня, могут выйти боком, так как как раз так этих 10-24 ЛП ему может не хватить на то что бы научится ковке у Беннета Erz-Schlachtklinge за 8 ЛП или буквально не докачать силу на 5, так как 5 силы уже не на что не влияют:-) но на самом деле влияют.

p.s.
385 и 390 урона в игре по большому счёту не надо. это уже из области эстетики :-) поэтому и "балансировку" Мечу Пала не очень понятно зачем делать

Название | урон | Бонус к 1р | требует | сила | урон | сила | урон
-- | -- | -- | -- | -- | -- | -- | --
Меч Пала | 140 | - | 100 силы | 245 | 385 | 250 | 390
Меч ОНД | 160 | 10% | 140 силы | 225 | 385 | 230 | 390

*****************************************

**часть 2** : северный тёмный

***************
_а почему же меч Пала такой слабый и зачем он такой слабый то нужен?_ 
он нужен для тех кто только познает мир Готики и не знает где какие плюшки раскиданы:-) После его освещения он сможет хотя бы с трех ударов  убивать Человека Ящера. и при таком раскладе(когда не получилось прокачать героя) он по своему лучше "El Bastardo",так как требует всего 100 силы, а имеет аж 120 урона.
**220 урона**

Название | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Gesegnete Erzklinge | 120 |   | 100 силы | 100 | 220
El Bastardo | 120 | 8% | 110 силы | 110 | 230

_Самокованный меч из руды 2 уровня, которому учит Беннет  имеет "всего" 100 урона и требует 80 силы. Кому он нужен в 3 главе?_ 
он нужен новичку. Новичок может выковать этот меч и наконец выйти в 3 главе на те самые 180 урона. после чего он начнёт "кайфовать" убивая шныгов/ползунов/снепперов иногда с 1 удара. Кстати новички в 3 главе после возвращения из Миненталя имеют часто уровень ниже, чем продвинутые игроки в 1 главе ;-) 
Поэтому я продублирую табличку из начала добавив туда этот самый меч

Название  | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Erz-Bastardschwert* | 100 | 10% | 80 силы | 80 | 180
Рапира | 100 | 8% | 50 ловк | 80 | 180
Machete | 100 | 3% | 90 силы | 90 | 190
Rubinklinge | 100 | 7% | 90 силы | 90 | 190
Bartaxt | 90 |   | 90 силы | 90 | 180
Doppelaxt | 90 |   | 90 силы | 90 | 180
Morgenstern | 90 |   | 90 силы | 90 | 180

*учит Беннет в 3 главе Наёмника

_Самокованный меч из руды 1 уровня, которому учит Беннет  это хлам. у него слишком маленький урон_. 
На самом деле это не так. Ведь если герой сможет его одеть(60 урона 10% к 1р, требует 40 силы), то он сможет иногда убивать  волка с 1 удара! а если ему сильно повезёт, то он сможет убить с трёх ударов луркера/ползуна/снеппера! 
_а падальщика?_
падальщика не сможет. но если поднимет силу ещё на 5, то тогда, иногда сможет.

Название | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Erz-Langschwert | 60 | 10 | 40 силы | 40 | 100
Kriegskeule | 50 |   | 50 силы | 50 | 100

_а полевых жуков?_
не сможет. для этого надо 120 урона, т.е. 60 силы. подкачается и сможет.

_но ведь в игре есть ещё.._
в игре много чего есть. об этом знает опытный игрок. что то можно украсть, что то отобрать, что то купить. но не опытный игрок не знает где какое оружие лежит. а для того что бы купить оружие 60/60 надо "много" золота(600 золотых). именно поэтому наверное игрок и пошёл учится ковке к Беннету, что бы "условно-бесплатно" выковать себе меч. и не простой меч, а меч из руды. который к тому же даёт большой бонус к 1р. Этот бонус наверняка порадует неопытного игрока, так как скорее всего он прокачивает одновременно и силу и 1р, немного туда, немного туда.

**********
> Шпага Мастера читерно дисбалансная

**********

Шпага Мастера необходимая часть баланса для того же мага или лучника, так как позволяет избить НПС не убивая его. Шпага Мастера никоим образом не влияет на баланс игры силачем с 1р: более высокий урон, фактически ни на что не влияет так как находится между порогами 180 и 230. Баланс, которого добились Пираньи стремится к идеальному, так как одновременно учитывает "возможности" начинающего игроку  и даёт интересные возможности опытному игроку. Понижения урона у той же Шпаги Мастера, может привести к тому что герой Маг не сможет этой Шпагой "пробить" броню ищущего. увеличения требований к ловкости, может привести к тому, что потребуется тратить ЛП на ловкость что бы её одет не ловкачу, что так же не совсем корректно.

Попытка дать возможность выковать хорошие мечи у Беннета раньше необходимого времени по факту просто ещё больше облегчит игру опытному игроку. особенно это касается одноручного клинка 140/120 , который позволит в 3 главе убивать с трёх ударов Ищущего, имея всего 95 силы. 124 силы это 95 силы +29 от амулета/пояса/колец.

Название | урон | Бонус к 1р | требует | сила | урон
-- | -- | -- | -- | -- | --
Шпага Мастера | 120 | 10% | 60 ловк | 144 | 264
El Bastardo | 120 | 8% | 110 силы | 144 | 264
Folteraxt | 125 | - | 125 силы | 139 | 264
Betty | 130 | - | 110 ловкости | 134 | 264
Orkschlächter | 130 | - | 130 силы | 134 | 264
Erz-Schlachtklinge* | 140 | 10% | 120 силы | 124 | 264

Разница в 20 урона эквивалентна 20 силе. 20 силы это 60-80 ЛП. разница в 10 силы и 10% это  55-80 ЛП.  т.е. 6-7 уровней. 
Перенос меча 100/80 в **начало** второй главу открывает альтернативную возможность игры: выходить на 180 урона в начале второй главы не прикладывая вообще никаких усилий :-)

**Comment by Kor-Angar** [comment]
> Основную возможность которую даёт Шпага Мастера это максимально рано выйти на высокий процент навыка 1р.

табличку сделал для наглядности. что бы в сравнение видна специфика игры через Шпагу Мастера(сила ГГ 60, Рапире(сила ГГ 80) и обычным силовым оружием(сила ГГ 90). 

Данные при условие что герой планирует качать **силу за ЛП до 90**, т.е. ещё не употребил бонусы к силе.

название   оружия | выход на 180 ущерба | 180 ущерба + докаченный 1р за ЛП | 180 ущерба + докаченный 1р за ЛП + скрижали 2 уровня
-- | -- | -- | --
Morgenstern / Machete / Rubinklinge /   Bartaxt / Doppelaxt | 14 уровень | 19 уровень |  21 уровень
Рапира | 15 уровень | 19 уровень | 19 уровень
Шпага Мастера | 17 уровень | 17 уровень | 17 уровень

Если качать **силу до 64 за ЛП** и начать употреблять бонусы к силе(молитвы)

название   оружия | выход на 180 ущерба | 180 ущерба + докаченный 1р за ЛП | 180 ущерба + докаченный 1р за ЛП + скрижали 2 уровня
-- | -- | -- | --
Morgenstern / Machete / Rubinklinge /   Bartaxt / Doppelaxt | 12 уровень | 17 уровень |  19 уровень
Рапира | 14 уровень | 17 уровень | 19 уровень
Шпага Мастера | 17 уровень | 17 уровень | 17 уровень

тут наглядно видно насколько Пираньи смогли ловко "сбалансировать" силовое и ловкостное оружие. Я думаю что для рядового игрока будет проще играть через классическое силовое оружие, без заморочек с собираньем всех бонусов ловкости. в таком случае рост героя будет более "плавный". Шпага Мастера позволяет совершить резкий "рывок". 
p.s.
на самом деле со Шпагой можно выйти на такие показатели ещё раньше, не на 17 уровне, а на 15 уровни, но для этого придётся приложить ещё больше усилия (забег в сокровищницу дракона, покупка пояса сила +10, собирание скрижалей на ловкость в ярике). Но это уже частные случаи и большая смахивает на "челлендж".

**Comment by Kor-Angar** [comment]
> Morgenstern / Machete / Rubinklinge / Bartaxt / Doppelaxt

блин. решил так поиграть, но не учёл важнейшего фактора: оружие убрали из продажи в 1 главе :-))  Machete правда оставили в продаже у Мартина+ второй мачете можно отобрать у охотника. И в пещере возле пирамиды есть Morgenstern, но там уже злой краулер-воин.

Самое печальное история с Bartaxt и Doppelaxt(хотел с ними поиграть), так как:
- Bartaxt визуально такой же как  у Шакала и Силаса в Готике 1.
- Doppelaxt визуально такой же как  у Лареса и Корда в Готике 1.


*****


>  180 ущерба + докаченный 1р за ЛП

в таблички уровни актуально для Наёма/ОнД. при игре за Мага/Пала, будет +3 уровня, так как им 1р качать надо больше. Это  я уже просто дошёл до 17 уровня, а одноручник не докачал и понял, что придётся ещё качать, так как герой маг :-)

**Comment by Kor-Angar** [comment]
> Силач не_мент сможет одеть во 2 главе Шпагу Мастера не вкладываю ЛП в ловкость, если пойдёт в Миненталь или Ярик за доп бонусами

смог поднять во второй главе не_ментом Шпагу Мастера в Хориносе

- 10 базовая ловкость
- 10 молитвы
- 1 бонус ренгару
- 9 эликсиры
- 6 скрижали
- 10 шмотки(пояс+кольцо)
- 14 ягод гоблина

Пришлось правда убить 11 паладинов в порту, а так же Элитного орка и варга(ради последней ягодки).
![image](https://user-images.githubusercontent.com/49514764/73144647-e64a2380-40b8-11ea-93db-e620d8513bc4.png)

всё таки наверное проще пойти в Ярик или Миненталь за доп ловкостью чем убивать Элитного Орка и устраивать Массовое убийство палов:-) так тому же что бы из Палов ягодки выпали надо ещё и настройку соответствующую включить.

**Comment by Kor-Angar** [comment]
в качестве примера взял в Ярик.
Шпага Мастера 120 урон + 85 силы = 205 ущерб 
Мачете 100 урон + 90 силы = 190 ущерб

Разницы не будет(хотя общий ущерб со шпагой фактически больше), так как кого то будет убивать с одного удара(луркеры, снепперы, краулеры), кого то с двух ударов, кого то с трёх.
https://www.youtube.com/watch?v=2woitkZrjFk
https://www.youtube.com/watch?v=YiytX3bq6b4
https://www.youtube.com/watch?v=Jif2IlUkka4

в тоже время конечно же очевидно, что на толстых монстрах, таких как например Демон или Тролль разница между Шпагой Мастера и Мачетой будет. и конечно же она будет в пользу Шпаги Мастера. Ровно как и на НПС(Паладинах,ОНД) это разница будет весьма заметна. 

со 180 так и будет идти до 220-230 ущерба. а это уже более высокий показатель силы и соответственно возможность взять силовое оружие с большим ущербом (110-120). При этом на 230 ущерба, силач который не тратил ЛП на ловкость выйдет быстрее, чем силач который потратил ЛП на ловкость что бы пораньше одеть Шпагу Мастера.

Ради интереса можно взять например цифру в 210 ущерба(120 шпага + 90 сила) и посмотреть на каком монстре она будет отличаться от 180. (90 оружие + 90 сила) в Ярике. На Огненном Мракорисе будет  разница. На ком ещё я навскидку не скажу. Но входе прохождения обязательно выясню :-)

**Comment by Kor-Angar** [comment]
https://worldofplayers.ru/threads/36817/page-191#post-1072633

**(Рапира 100+20 силы) = (Оружие 60+60 силы) = 120**

Что качаем | ЛП | Что качаем | ЛП | Примечание
-- | -- | -- | --| --
1р | 24 | 1 р | 24 | .
Ловкость | 12 | . | . | .
. | . | Сила до 45 | 46 | .
скрижали | 15 | . | . | .
взлом | 10 | взлом | 10 | .
воровство | 10 | . | . | .
итого | 71 | итого | 80 | .

. | . | . | . 
-- | -- | -- | --
1р | 34 | 1р | 34
Бонус к 1р | 8 | Бонус к 1р | 5
Чистая сила | 10 | Чистая сила | 42
Чистая ловкость | 24 | Чистая ловкость | 10
Амуле силы | 10 | Амуле силы | 10
Кольцо ловоксти | 5 | . | .
Пояс защиты(?) | . | Пояс силы | 5


**120 урона это** убийство волков, падальщиков, крыс, гоблинов, черных гоблинов, полевых жуков с одного удара
Луркеров, краулеров и снепперов с двух ударов.

********

Что бы убивать:

- [Луркеров, Краулеров, Снепперов, Beisser, Sumpfratte, Graslandscavenger  с одного удара](http://mondgesaenge.de/G2ADB/guide_monster.htm#section_3)
- [Варанов, Краулеров Войнов, Bluthund, Fangheuschrecke, Razor](http://mondgesaenge.de/G2ADB/guide_monster.htm#section_3)  с двух ударов 

**нужно уже 180 урона.**

_Впрочем и тут есть кое какие нюансы.. например 150 урона позволяет убивать  Кабана с одного удара, а Fangheuschrecke с двух.  Что бы убивать ящеров с двух ударов достаточно иметь 160 урона._

**(Рапира 100+80 силы) = (Оружие 90+90 силы) = (Шпага Мастера 120 + 60 силы) = 180**


при этом что бы одеть Шпагу Мастера придётся уходить в Ярик или Миненталь за доп бонусами к ловкости или потратить ЛП на ловкость(в табличке ниже уровень  для одевание Шпаги /Рапиры считался без траты ЛП на ловкость).

*****

![image](https://user-images.githubusercontent.com/49514764/73180628-bf2a3b00-4126-11ea-950c-33c83ce0f8fc.png)


*****

в целом конечно через Шпагу играть комфортнее, так как не надо заморачиваться. Собирание бонусов к ловкости, оправдывает изучения воровства :-) при игре через Шпагу раньше берутся языки и можно читать скрижали на жизни/ману  с 1 главы. Ранний выход на 1р=100%. Неоспоримые преимущества есть, но есть и свои нюансы связанные с собиранием бонусов и дополнительным игровым временем в начале игры на те же молитвы.  
Лично мне через Шпагу нравится играть больше. Намного:-) Ранний выход на 100% 1р для меня в приоритете и Шпага это позволяет сделать. Но это вовсе не значит что Шпага/Рапира дисбалансные:-)
Но тут следует справедливости ради отметить что при прокачке героя под классическое силовое оружием будет более ранний выход(в сравнение с шпагой) на 220 / 230 / 250 /260 ущерба(110-130 силы). А [дробящее оружие позволит так же убивать каменных стражей с одного удара, а големов с двух](https://www.youtube.com/watch?v=ualhwYdAB2I), что при прокачки силовика под Шпагу сделать не получится:-) Такое вот открытие я сделал для себя:-)

**Comment by Kor-Angar** [comment]
Исправил информацию касательно освященного одноручного клинка Паладинов
![IMG_20200131_162140](https://user-images.githubusercontent.com/49514764/73542720-691b0780-4446-11ea-83d0-51218648bec1.png)
Огромное спасибо Alex007 за то что внимательно проверил данные в таблице и нашёл грубейшую ошибку в расчётах! 
https://worldofplayers.ru/threads/36817/post-1073135

**Comment by Kor-Angar** [comment]
если в очередной раз будет подниматься вопрос о правках меча паладина.
в своей сборке его поправил  вот так

  | Было | стало
-- | -- | --
Базовая версия (3 глава) | 100 / 100 | 120 / 100
Освященная версия  (3-4 глава) | 120 / 100 | 130 / 100
Финальная версия  (4-5 глава) | 140 / 100 | 140 / 100



таким образом финальная версия осталась такой же, а у предыдущих версия поднят урон. в первую очередь для того что бы освященный меч паладина не был хуже Orkschlächter 130 / 130, так как в таком случае Паладин будет ходить с мечём Orkschlächter на битву с Драконом, что с точки зрения ЛОРа неправильно. при этом такой шаг не затрагивает баланс так ущерб у меча не больше чем у Orkschlächter , который можно найти в первой главе. При этом мы видим опять же с точки зрения ЛОРа освященный меч паладина из руды действительно лучше так как требует меньше силы, чем Orkschlächter .


****


Было | стало
-- | --
Базовая версия 100 / 0% / 100 |  
Rubinklinge 100 / 7% / 90 | Rubinklinge 100 / 7% / 90
ItMw_Schwert5 110 / 0% / 110 | ItMw_Schwert5 110 / 0% / 110
El Bastardo 120 / 8% / 110 | El Bastardo 120 / 8% / 110
Освященная версия 120 / 0% / 100 | Базовая версия 120 / 0% / 100
Folteraxt 125 / 0% / 125 | Folteraxt 125 / 0% / 125
Orkschlächter 130 /  0% / 130 | Orkschlächter 130 /  0% / 130
  | Освященная версия 130 / 0% /  100



Rubinklinge - получает мент бесплатно начало 3 главы
Базовая версия меч пала - покупает Пал за 2000 в 3 главе у кузнеца
El Bastardo  - учит кузнец

**Reply by Kor-Angar** [reply]
эта информация важная

**Reply by dosinabox** [reply]
> эта информация важная

Не спорю, просто переношу в обсуждения. Раздел Issues немного для другого.

**Reply by Kor-Angar** [reply]
я понял :-) просто акцентировал на это внимание :-)

**Comment by Kor-Angar** [comment]
Запости сюда плиз как будет время твои правки по оружию которые выдаёт Пек менту. Так же вопрос вытекает такой неочивидный: есть ли условия при которых  Пек даст рубиновый меч Паладину?
Я помню там был скользкий момент, что пока сидит Беннет надо будучи ментом не забыть поговорить с Пеком и Гриттой.

**Reply by Kor-Angar** [reply]
Гритта опыт даёт, пока Беннет сидит, этот момент тоже можно перепроверить.

**Reply by dosinabox** [reply]
> Так же вопрос вытекает такой неочивидный: есть ли условия при которых Пек даст рубиновый меч Паладину?

Это невозможно, потому что на каждом случае получения рубинового клинка стоит условие `other.guild == GIL_MIL`.

**Reply by dosinabox** [reply]
```cpp
func int DIA_Gritta_Perm3U4U5_Condition()
{
	return TRUE;
};

func void DIA_Gritta_Perm3U4U5_Info()
{
	AI_Output(other,self,"DIA_Gritta_Perm3U4U5_15_00");	//Как дела?
	if(Kapitel == 3)
	{
		if(MIS_RescueBennet != LOG_SUCCESS)
		{
			AI_Output(self,other,"DIA_Gritta_Perm3U4U5_16_01");	//Все здесь катится в тартарары. Куда не взглянешь, везде одни преступления и насилие. Только представь, недавно убили даже паладина!
			Info_ClearChoices(DIA_Gritta_Perm3U4U5);
			Info_AddChoice(DIA_Gritta_Perm3U4U5,Dialog_Back,DIA_Gritta_Perm3U4U5_BACK);
			Info_AddChoice(DIA_Gritta_Perm3U4U5,"Эта война - ради вас. Такое случается.",DIA_Gritta_Perm3U4U5_War);
			Info_AddChoice(DIA_Gritta_Perm3U4U5,"Все будет хорошо.",DIA_Gritta_Perm3U4U5_TurnsGood);
			Info_AddChoice(DIA_Gritta_Perm3U4U5,"Что ты слышала?",DIA_Gritta_Perm3U4U5_Rumors);
		}
		else
		{
			AI_Output(self,other,"DIA_Gritta_Perm3U4U5_16_02");	//Ох, я так взволнована. Лорд Хаген освободил этого наемника.
			AI_Output(other,self,"DIA_Gritta_Perm3U4U5_15_03");	//А какое это имеет отношение к тебе?
			AI_Output(self,other,"DIA_Gritta_Perm3U4U5_16_04");	//Да ты сам подумай. Ты веришь, что наемники сидели бы сложа руки, когда их товарища собирались повесить?
			AI_Output(self,other,"DIA_Gritta_Perm3U4U5_16_05");	//Они бы попытались освободить его, и пролилось бы много крови. Но, слава Инносу, все разрешилось удачно.
		};
	}
	else if(Kapitel == 5)
	{
		AI_Output(self,other,"DIA_Gritta_Perm3U4U5_16_06");	//Паладины к чему-то готовятся. Похоже, они собираются уходить из города.
	}
	else
	{
		AI_Output(self,other,"DIA_Gritta_Perm3U4U5_16_07");	//Все как всегда, но я не хочу жаловаться.
	};
};

func void DIA_Gritta_Perm3U4U5_BACK()
{
	Info_ClearChoices(DIA_Gritta_Perm3U4U5);
};

func void DIA_Gritta_Perm3U4U5_War()
{
	AI_Output(other,self,"DIA_Gritta_Perm3U4U5_War_15_00");	//Эта война - ради вас. Такое случается.
	AI_Output(self,other,"DIA_Gritta_Perm3U4U5_War_16_01");	//Да, всему виной эта ужасная война. Все страдают от нее. Не знаю даже, как им удается сводить концы с концами.
	AI_Output(self,other,"DIA_Gritta_Perm3U4U5_War_16_02");	//Иногда я спрашиваю себя, что мы сделали, чтобы заслужить такое наказание от Инноса.
};

func void DIA_Gritta_Perm3U4U5_TurnsGood()
{
	AI_Output(other,self,"DIA_Gritta_Perm3U4U5_TurnsGood_15_00");	//Все будет хорошо.
	AI_Output(self,other,"DIA_Gritta_Perm3U4U5_TurnsGood_16_01");	//Ты пытаешься подбодрить меня. Это мило.
	if(GrittaXP_Once == FALSE)
	{
		B_GivePlayerXP(XP_AmbientKap3);
		GrittaXP_Once = TRUE;
	};
};

func void DIA_Gritta_Perm3U4U5_Rumors()
{
	AI_Output(other,self,"DIA_Gritta_Perm3U4U5_Rumors_15_00");	//Что ты слышала?
	AI_Output(self,other,"DIA_Gritta_Perm3U4U5_Rumors_16_01");	//Только то, что говорят на улицах.
	AI_Output(self,other,"DIA_Gritta_Perm3U4U5_Rumors_16_02");	//Я слышала, что убийцу уже поймали, но я не знаю, правда это или нет.
};
```
